tests/worker/test_profile.py

```diff
@@ -46,20 +42,24 @@ def mock_mem_info():
 
     from unittest.mock import patch
     with patch("torch.cuda.mem_get_info", side_effect=mock_mem_info):
+        # Load the model so we can profile it
+        worker.init_device()
+        worker.load_model()
         gpu_blocks, _ = worker.determine_num_available_blocks()
 
-    # Peak vram usage by torch should be 0.7077 GiB
```

Comment:
this peak memory usage is actually wrong. it is the peak memory plus model weights, `0.71 ≈ 0.47 + 0.25` .

https://buildkite.com/vllm/fastcheck/builds/9871#0193c3b5-31ef-4372-b0d0-55c643842f1a/198-8536 shows the peak memory is 

>  [INFO 12-13 22:56:44 worker.py:243] PyTorch activation peak memory	0.47GiB

Reply:
Yeah I think this is just a language problem. 0.707GB is the peak if you measure from before the model loads, 0.47 is the peak measuring from after the model loads. As long as we agree to pick one meaning and stick with it it's fine, and we can use how you've defined it here 👍